Задание № 1 - готово

lenysi4ka25 opened the pull request:
Домашнее задание № 1 и 2 - исправила!

lenysi4ka25 committed:
Задание № 1 - готово

lenysi4ka25 committed:
Задание № 1 и № 2 - готово
Задание № 3

lenysi4ka25 committed:
Задание № 1 и № 2 - готово
Задание № 3 и № 4 - готово

lenysi4ka25 commented:
Задание № 3 и № 4 - готово

iStemmer commented on src/Bus.java:
```diff
@@ -52,20 +52,38 @@ public String toString() {
+    public String bestLapTime() {
         System.out.println(getMarka() + " - лушее время круга");
-        return new String[0];
+        return null;
```
лучше null не возвращать, или вернуть getMarka() + " - лушее время круга"

lenysi4ka25 replied:
public String bestLapTime() {
        System.out.println(getMarka() + " - лушее время круга");
        return getMarka();
    }

iStemmer commented on src/Bus.java:
```diff
@@ -52,20 +52,38 @@ public String toString() {
+    public int maxSpeed() {
         System.out.println(getMarka() +  " - максимальная скорость");
-        return new int[0];
+        return 0;
```
нет смысла в методах которые возвращают 0 или null
Пусть возвращают то, что распечатывают

lenysi4ka25 replied:
public void maxSpeed() {
        System.out.println(getMarka() +  " - максимальная скорость");
    }

iStemmer commented on src/Car.java:
```diff
@@ -54,20 +54,38 @@ public String toString() {
+    public String pitStop() {
         System.out.println(getMarka() + " - заехать на Пит-Стоп");
-        return new String[0];
+        return null;
```
возвращаем что-то иди делаем их void

lenysi4ka25 replied:
public String pitStop() {
        return getMarka() + " - заехать на Пит-Стоп";
    }

iStemmer commented on src/Car.java:
```diff
@@ -54,20 +54,38 @@ public String toString() {
+    public String bestLapTime() {
         System.out.println(getMarka() + " - лушее время круга");
-        return new String[0];
+        return null;
```
не возвращаем пустоту

lenysi4ka25 replied:
public String bestLapTime() {
        return getMarka() + " - лушее время круга";
    }

lenysi4ka25 committed:
Задание № 1 и № 2 - готово
Задание № 3 и № 4 - готово
Внесла изменения.

lenysi4ka25 committed:
Задание № 1 и № 2 - готово
Задание № 3 и № 4 - готово
Внесла изменения.
Добавила исключения

lenysi4ka25 committed:
Задание № 1 и № 2 - готово
Задание № 3 и № 4 - готово
Внесла изменения.
Добавила исключения
в класс транспорт добавила поле "список водителей"

lenysi4ka25 committed:
Задание № 1 и № 2 - готово
Задание № 3 и № 4 - готово
Внесла изменения.
Добавила исключения
в класс транспорт добавила поле "список водителей"
создала клас "Sponsor"

lenysi4ka25 committed:
Задание № 1 и № 2 - готово
Задание № 3 и № 4 - готово
Внесла изменения.
Добавила исключения
в класс транспорт добавила поле "список водителей"
создала клас "Sponsor"
создала класс "Mechanic" и внесла необходмые данные в другие классы)

lenysi4ka25 committed:
Задание № 1 и № 2 - готово
Задание № 3 и № 4 - готово
Внесла изменения.
Добавила исключения
в класс транспорт добавила поле "список водителей"
создала клас "Sponsor"
создала класс "Mechanic" и внесла необходмые данные в другие классы)
Выводит водителя, механика и спонсора

lenysi4ka25 committed:
Задание № 1 и № 2 - готово
Задание № 3 и № 4 - готово
Внесла изменения.
Добавила исключения
в класс транспорт добавила поле "список водителей"
создала клас "Sponsor"
создала класс "Mechanic" и внесла необходмые данные в другие классы)
Выводит водителя, механика и спонсора
добавила "сумму спонсорской помощи"

lenysi4ka25 committed:
Задание № 1 и № 2 - готово
Задание № 3 и № 4 - готово
Внесла изменения.
Добавила исключения
в класс транспорт добавила поле "список водителей"
создала клас "Sponsor"
создала класс "Mechanic" и внесла необходмые данные в другие классы)
Выводит водителя, механика и спонсора
добавила "сумму спонсорской помощи"
Вывела на печать через toString

lenysi4ka25 committed:
Задание № 1 и № 2 - готово
Задание № 3 и № 4 - готово
Внесла изменения.
Добавила исключения
в класс транспорт добавила поле "список водителей"
создала клас "Sponsor"
создала класс "Mechanic" и внесла необходмые данные в другие классы)
Выводит водителя, механика и спонсора
добавила "сумму спонсорской помощи"
Вывела на печать через toString
Внесла небольшие правки

lenysi4ka25 committed:
Задание № 1 и № 2 - готово
Задание № 3 и № 4 - готово
Внесла изменения.
Добавила исключения
в класс транспорт добавила поле "список водителей"
создала клас "Sponsor"
создала класс "Mechanic" и внесла необходмые данные в другие классы)
Выводит водителя, механика и спонсора
добавила "сумму спонсорской помощи"
Вывела на печать через toString
Внесла небольшие правки
Добавила класс "Техобслуживание" и в нем метод проверки

lenysi4ka25 committed:
Задание № 1 и № 2 - готово
Задание № 3 и № 4 - готово
Внесла изменения.
Добавила исключения
в класс транспорт добавила поле "список водителей"
создала клас "Sponsor"
создала класс "Mechanic" и внесла необходмые данные в другие классы)
Выводит водителя, механика и спонсора
добавила "сумму спонсорской помощи"
Вывела на печать через toString
Внесла небольшие правки
Добавила класс "Техобслуживание" и в нем метод проверки
Выполнила домашнее задание № 3 (HashMap).

lenysi4ka25 committed:
Задание № 1 и № 2 - готово
Задание № 3 и № 4 - готово
Внесла изменения.
Добавила исключения
в класс транспорт добавила поле "список водителей"
создала клас "Sponsor"
создала класс "Mechanic" и внесла необходмые данные в другие классы)
Выводит водителя, механика и спонсора
добавила "сумму спонсорской помощи"
Вывела на печать через toString
Внесла небольшие правки
Добавила класс "Техобслуживание" и в нем метод проверки
Выполнила домашнее задание № 3 (HashMap).
Домашнее задание 5 из урока от 14.11.2022 года.

lenysi4ka25 committed:
Задание № 1 и № 2 - готово
Задание № 3 и № 4 - готово
Внесла изменения.
Добавила исключения
в класс транспорт добавила поле "список водителей"
создала клас "Sponsor"
создала класс "Mechanic" и внесла необходмые данные в другие классы)
Выводит водителя, механика и спонсора
добавила "сумму спонсорской помощи"
Вывела на печать через toString
Внесла небольшие правки
Добавила класс "Техобслуживание" и в нем метод проверки
Выполнила домашнее задание № 3 (HashMap).
Домашнее задание 5 из урока от 14.11.2022 года.
"Итераторы"
